pkg/controller/operators/catalog/testdata/prometheusrule.cr.yaml

```diff
@@ -11,6 +11,8 @@ spec:
       rules:
         - alert: SeriousAlert 
           annotations:
+            summary: Serious Alert!
+            description: This is a very serious alert!
```

Comment:
Should we get rid of it all together or just change the copy? I was thinking it may help with consistency across the other alerts even if it is just a test.